lib/logatron/logatron.rb

```diff
@@ -72,5 +74,14 @@ def msg_id
     def msg_id=(id)
       Logatron::Contexts.msg_id = id
     end
+
+    private
+
+    def exception_message(e, additional)
+      info = additional.is_a?(Hash) ? additional.map { |(k, v)| "#{k}=>#{v}"}.join(', ') : additional
+      info_msg = info.nil? || info.empty? ? '' : "; MORE_INFO( #{info} )"
+      backtrace = configuration.backtrace_cleaner.clean(e.backtrace).join(' -> ')
+      "#{e.class} #{e.message}#{info_msg} -> #{backtrace}"
```

Comment:
I thought of that too, however backtraces are extremely noisy. I thought putting anything after the backtraces would be lost. As the code now stands, the thinking is that the additional info is part of the message.